tests/test_monitoring.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from test_helpers import (
+    get_loki_labels, get_prometheus_labels, verify_loki_labels, verify_prometheus_labels
+)
+
+@pytest.mark.order(8)
+def test_monitoring_labels():
+    print("\n=== Testing Monitoring Labels ===")
+
+    loki_labels = get_loki_labels()
+    verify_loki_labels(loki_labels)
```

Comment:
The verify function IS the test. As mentioned above I recommend splitting this into two tests. One for Loki and one for Prometheus. And then implement each of them as an actual test which means failing them if the labels aren't as expected.